src/components/loading-skeleton.tsx

```diff
@@ -1,8 +1,10 @@
+import React from 'react';
+
 function randomNumber(min: number, max: number) {
   return Math.floor(Math.random() * (max - min + 1)) + min;
 }
 
-const LoadingSkeleton = () => {
+const LoadingSkeleton: React.FC = () => {
```

Comment:
Is this repo just using Preact or is mixed with React?

Reply:
Good point. My mistake. I'll replace with preact correspondent component.